wger/core/models.py

```diff
@@ -207,6 +222,14 @@ class UserProfile(models.Model):
                        validators=[MinValueValidator(10), MaxValueValidator(100)])
     '''The user's age'''
 
+    birthdate = models.DateField(verbose_name=('Date of Birth'),
+                                 help_text=_('Please use the format: mm/dd/yyyy'),
```

Comment:
Better not, it seems there are still some browser compatibility issues with date input elements, just leave it as it is